simple-service-webapp-tests/src/test/java/com/example/TestTest.java

```diff
@@ -0,0 +1,132 @@
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Test
+    public void testPostMethod() throws IOException {
+
+
+        HttpTransport transport = new NetHttpTransport();
+        HttpRequestFactory requestFactory = transport.createRequestFactory();
+
+        GenericUrl url = new GenericUrl("http://localhost:8080/api/userstories");
+
+
+        UserStory userStory = new UserStory(10, "newus", 10);
+        System.out.println(mapper.writeValueAsString(userStory));
+        String token = "Bearer "+authenticate();
+        HttpRequest request =
+                requestFactory.buildPostRequest(url, new ByteArrayContent("application/json", mapper.writeValueAsString(userStory).getBytes()))
+                        .setHeaders(new HttpHeaders().set("Authorization", Collections.singletonList(token)));
+
+        String response = request.execute().parseAsString();
+        //System.out.println(response);
+
+    }
+
+    @Test
+    public void testGetMethod() throws IOException {
+
+
+        HttpTransport transport = new NetHttpTransport();
+        HttpRequestFactory requestFactory = transport.createRequestFactory();
+
+        GenericUrl url = new GenericUrl("http://localhost:8080/api/userstories");
+        String token = "Bearer "+authenticate();
+        HttpRequest request =
+                requestFactory
+                        .buildGetRequest(url)
+                        .setHeaders(new HttpHeaders()
+                                .set("Authorization", Collections.singletonList(token))
+                        );
+
+        Collection<UserStory> userStories = mapper.readValue(request.execute().parseAsString(), new TypeReference<List<UserStory>>(){});
+
+        for(UserStory us : userStories){
+            System.out.println(us.toString());
+        }
+
+    }
+
+    @Test
+    public void testDeleteMethod() throws IOException {
+        HttpTransport transport = new NetHttpTransport();
+        HttpRequestFactory requestFactory = transport.createRequestFactory();
+
+        GenericUrl url = new GenericUrl("http://localhost:8080/api/userstories/10");
+        String token = "Bearer "+authenticate();
+        HttpRequest request =
+                requestFactory
+                        .buildDeleteRequest(url)
+                        .setHeaders(new HttpHeaders()
+                                .set("Authorization", Collections.singletonList(token))
+                        );
+        System.out.println(request.execute().parseAsString());
+    }
+
+    @Test
+    public void testPutMethod() throws IOException {
+        HttpTransport transport = new NetHttpTransport();
+        HttpRequestFactory requestFactory = transport.createRequestFactory();
+
+        GenericUrl url = new GenericUrl("http://localhost:8080/api/userstories/10");
+        String token = "Bearer "+authenticate();
+
+        UserStory userStory = new UserStory(10, "newus 22", 1011);
```

Comment:
Ok, let's fix the rest api by making it generate the id for a new object, the id in the user story sent should be ignored.
The server should generate a random unique id for every new user story.